website/docs/references/models-http-api/voyage-ai.md

```diff
@@ -1,12 +1,14 @@
 # Voyage AI
 
-[Voyage AI](https://voyage.ai/) is a company that provides a range of embedding models. Tabby supports Voyage AI's models for embedding tasks.
+[Voyage AI](https://voyage.ai/) is a company that specializes in developing high-performance embedding models, particularly optimized for code understanding and processing tasks.
```

Comment:
`embedding tasks` should not be changed to `code understanding and processing tasks`